static/js/popovers.js

```diff
@@ -519,10 +521,14 @@ exports.register_click_handlers = function () {
         if (emoji_map_is_open) {
             // If the popover is already shown, clicking again should toggle it.
             popovers.hide_emoji_map_popover();
-            return;
+        } else { // popovers.hide_all();
```

Comment:
Yeah. I kept it so that it would remind me to check areas like this is where there has been unwanted calls to hide all popovers. And understand them. so that i could optimize other areas like this as well.  I forgot to remove it later.